e2e/scenario_test.go

```diff
@@ -649,6 +653,7 @@ func Test_ubuntu2204AirGap(t *testing.T) {
 				nbc.ContainerService.Properties.AgentPoolProfiles[0].Distro = "aks-ubuntu-containerd-22.04-gen2"
 				nbc.AgentPoolProfile.Distro = "aks-ubuntu-containerd-22.04-gen2"
 
+				// TODO: define below in the cluster config instead of mutate bootstrapConfig
```

Comment:
nit: could you add TODO (name): so that other maintainers in the repo know who is looking into it

Reply:
added thx